client/src/annoMatrix/loader.ts

```diff
@@ -327,13 +327,13 @@ export default class AnnoMatrixLoader extends AnnoMatrix {
 Utility functions below
 */
 
-function _writableCheck(colSchema: ColumnSchema) {
+function _writableCheck(colSchema: ArraySchema): void {
   if (!colSchema || !("writable" in colSchema && colSchema.writable)) {
     throw new Error("Unknown or readonly obs column");
   }
 }
 
-function _writableCategoryTypeCheck(colSchema: ColumnSchema): void {
+function _writableCategoryTypeCheck(colSchema: ArraySchema): void {
```

Comment:
see type signature change suggestion on `_writableCheck`

Reply:
Udpated [_writableObsCheck](https://github.com/chanzuckerberg/cellxgene/blob/mim/annoMatrix/client/src/annoMatrix/loader.ts#L346) and [_writableObsCategoryTypeCheck](https://github.com/chanzuckerberg/cellxgene/blob/mim/annoMatrix/client/src/annoMatrix/loader.ts#L352).